src/components/molecules/NavItem/index.stories.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import { ComponentStory, ComponentMeta } from '@storybook/react'
+import NavItem from '.'
+
+export default {
+  title: 'molecules/NavItem',
+  component: NavItem,
+  argTypes: {
+    iconName: {
+      control: {
+        type: 'select',
+        options: [
+          'home',
+          'clientManagement',
+          'configurations',
+          'accessManagement',
+          'orchesstration',
+          'riskCaseManagement',
+          'reports',
+        ],
+        default: 'home',
+      },
+    },
+    text: {
+      control: {
+        type: 'select',
+        options: [
+          'Home',
+          'Client Management',
+          'Configurations',
+          'Access Management',
+          'Orchesstration',
+          'Risk Case Management',
+          'Reports',
+        ],
+        default: 'Home',
+      },
+    },
+    active: {
+      control: {
+        type: 'select',
+        options: [true, false],
+        default: false,
+      },
+    },
+    route: {
+      control: {
+        type: 'select',
+        options: [
```

Comment:
Currently NavItem is configured for 6 icons? 

If a new icon is introduced, How to include that in NavItem how to achieve that scenario?

Reply:
if we want to have a new icon in NavItem ,we can pass it as prop to icon atom ,here those 6 icons are for user experience to select from one from drop down in storybook.  So if we have a new icon we just need to add it to Constants.ts file..